app/helpers/application_form_helper.rb

```diff
@@ -0,0 +1,35 @@
+module ApplicationFormHelper
+  def application_form_full_name(application_form)
+    "#{application_form.given_names} #{application_form.family_name}"
+  end
+
+  def application_form_summary_rows(application_form)
+    [
+      [
+        I18n.t("application_form.summary.country"),
+        CountryName.from_country(application_form.region.country)
+      ],
+      [I18n.t("application_form.summary.region"), application_form.region.name],
+      [
+        I18n.t("application_form.summary.created_at"),
+        application_form.created_at.strftime("%e %B %Y")
+      ],
+      [I18n.t("application_form.summary.days_remaining_sla"), "17"],
```

Comment:
I wonder if we should set this to 'Not implemented' or something like that as I expect this might not get looked at for at least a few sprints and it'll look like we've implemented it at a casual glance if we use a number.

Reply:
Sounds reasonable, I'll update it!